spec/tapioca/compilers/symbol_table_compiler_spec.rb

```diff
@@ -57,7 +57,11 @@ def ruby_version(selector)
     end
 
     def template(src)
-      ERB.new(src, nil, ">").result(binding).chomp
+      if RUBY_VERSION >= '2.6'
```

Comment:
Thanks for catching this. Can we do the check like [here](https://github.com/rtomayko/tilt/commit/4aff2d124e684c40240baf2d79d0ca1cd4cfc0f0) instead of relying on Ruby version?

Reply:
Did the 2 changes as requested. TravisCI is green.